### PR TITLE
Improve blocking instance UX

### PR DESCRIPTION
**Pull request by maximemenager**

### CODE REVIEW

File: web/middlewares/instance.go [hunk 1]
```diff
@@ -63,12 +64,11 @@ func CheckInstanceBlocked(next echo.HandlerFunc) echo.HandlerFunc {
...
 			// Fallback by trying to determine the blocking reason
 			reason := i.BlockingReason
-			returnCode := http.StatusPaymentRequired
 
 			if reason == "" {
-				reason = http.StatusText(http.StatusPaymentRequired)
+				reason = i.Translate(instance.BlockedUnknown.Message)
 			} else if reason == instance.BlockedPaymentFailed.Code {
-				reason = instance.BlockedPaymentFailed.Message
+				reason = i.Translate(instance.BlockedPaymentFailed.Message)
```
Comment (nono) [inline web/middlewares/instance.go]: Shouldn't we have `returnCode = http.StatusPaymentRequired` here?

File: web/statik/statik.go [hunk 2]
```diff
@@ -488,6 +488,363 @@ jy/+fwAAAP//TfvNpxmuAQA=
...
+zstA3AhpcNjIee5FYVm9oYFWyX0vC7vO+QvqNG49m9TjeT+Mk4fmwxcpb/KWvTFS
+PylD/hefOA/d+BcZ8DdOLQKYNh4JwHlHw61Awd84NQgT3vD98GhbZSaRBMvg88Wt
+eFX/lojfMvF7cMEzhj6XqUEzzrDsJLEj5H4B+Y6/FsRL/vUd4Er9vUa+nd0LrMQ4
+xCbhyqxD7j6A5YfWvU2TovUK0Qfg4FygWhQ2++yyI58fNNM4wMn0cMGnSeq6qk55
+/IrkQ8MvAeegjSKZhRJwHhb4DwyPEZhbLA9kdpHUqY8EyRarIy/FV1zfpfZou5H1
+PrLt4FUPwFJ/wHsjcMI+MOX+MH1IPTfefkyjg2/3AT7oD5x5ltcNedgfspGnUS+Y
+N/1hYvuDuzR49fh/AwAA//9Qx9WryDMBAA==
+-----END COZY ASSET-----
+-----BEGIN COZY ASSET-----
+Name: /css/cozy-ui.min.css
```
Comment (nono) [inline web/statik/statik.go]: I don't think it is normal to have `/css/cozy-ui.min.css` twice!

File: web/middlewares/instance.go (hunk 1 above)
Comment (maximemenager) [reply web/middlewares/instance.go]: Thanks, fixed ;)

File: web/statik/statik.go (hunk 2 above)
Comment (maximemenager) [reply web/statik/statik.go]: Nice pick, fixed !

**Pull request merged**